src/Microsoft.TestPlatform.Extensions.BlameDataCollector/Resources/xlf/Resources.cs.xlf

```diff
@@ -62,7 +62,7 @@
       </trans-unit>
       <trans-unit id="UnexpectedValueForInactivityTimespanValue">
         <source>Unexpected value '{0}' provided as timeout. Please provide a value in this format: 1.5h / 90m / 5400s / 5400000ms / 5400000</source>
-        <target state="new">Neočekávaná hodnota {0} zadaná pro ExpectedExecutionTimeOfLongestRunningTestInMinutes. Zadejte jako vstup kladné celé číslo.</target>
+        <target state="new">Unexpected value '{0}' provided as timeout. Please provide a value in this format: 1.5h / 90m / 5400s / 5400000ms / 5400000</target>
```

Comment:
Those strings are not translated, is that expected?

Reply:
String state is new and we will cover them in next  delivery.

Reply:
PR updated with fully loclaized xlf